samples/sampler/stories/default/Image.js

```diff
@@ -5,9 +5,9 @@ import Image, {ImageBase, ImageDecorator} from '@enact/sandstone/Image';
 import {ImageBase as UiImageBase} from '@enact/ui/Image';
 
```

Comment:
```suggestion
import {svgGenerator} from '../helper/svg';
```
As svg generator code is duplicated in sampler and qa-sampler, it would be better to place the svgGenerator function in the helper folder.